site/docs/v1/tech/apis/identity-providers/links.adoc

```diff
@@ -60,7 +63,7 @@ include::docs/v1/tech/apis/identity-providers/_links-post-response-body.adoc[]
 
 == Complete a pending Link
 
-This API is used complete a pending Link. If an identity provider is configured with a linking strategy of `Create a pending link`, a pending link Id will be returned from the Login API that can be used here.
+This API is used complete a pending Link. If an identity provider is configured with a linking strategy of `Create a pending link`, a pending link Id will be returned from the Login API for the particular IdP (`POST /api/identity-provider/login`) can be used here.
```

Comment:
Can you read this sentence aloud? The last 4 words don't make sense to me in the context of the entire sentence. Can we split it up?